Adding GenTraversableLike to whitelist breaks compilation of GenMapLike

```scala
./scala-scala/src/library/scala/collection/GenMapLike.scala:119: error: forall is not a member of scala.collection.GenMapLike[A, B, Repr](GenMapLike.this)
        this forall {
        ^
```

--- a/src/dotty/tools/dotc/core/SymDenotations.scala
+++ b/src/dotty/tools/dotc/core/SymDenotations.scala
@@ -1487,10 +1487,8 @@ object SymDenotations {
     def delete(sym: Symbol)(implicit ctx: Context) = {
       require(!(this is Frozen))
       info.decls.openForMutations.unlink(sym)
-      if (myMemberFingerPrint != FingerPrint.unknown)
-        computeMemberFingerPrint
-      if (myMemberCache != null)
-        myMemberCache invalidate sym.name
+      myMemberFingerPrint = FingerPrint.unknown
+      if (myMemberCache != null) myMemberCache invalidate sym.name
     }
 
     /** All members of this class that have the given name.

--- a/test/dotc/scala-collections.whitelist
+++ b/test/dotc/scala-collections.whitelist
@@ -141,7 +141,7 @@
 ./scala-scala/src/library/scala/collection/GenSetLike.scala
 
 # breaks genMapLike https://github.com/lampepfl/dotty/issues/937
-#./scala-scala/src/library/scala/collection/GenTraversable.scala
+./scala-scala/src/library/scala/collection/GenTraversable.scala
 ./scala-scala/src/library/scala/collection/GenTraversableLike.scala
 
 ./scala-scala/src/library/scala/collection/GenTraversableOnce.scala

